definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState-pcq-nonprod.json

```diff
@@ -180,5 +180,61 @@
     "CaseStateID": "AosCompleted",
     "UserRole": "caseworker-divorce-pcqextractor",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "03/07/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "allowPetitionerToAmend",
```

Comment:
This will need to be "CaseStateID"

Reply:
Great catch. That was a mistake i didnt even notice. Cheers